post: 2024.12 announcement (#6)
grml/blog.grml.org: content/post/418-Grml-new-stable-release-2024.12-available.md

---
author: Michael Prokop
categories: []
date: Fri, 20 Dec 2024 11:42:23 +0100
layout: post
slug: 418-Grml-new-stable-release-2024.12-available
title: Grml - new stable release 2024.12 available

---

> We are proud to announce the new stable release version 2024.12, code-named ['Adventgrenze'](https://grml.org/faq/#releasename)!

Reviewer: ```suggestion
We are proud to announce our new stable release version 2024.12, code-named ['Adventgrenze'](https://grml.org/faq/#releasename)!
```

> This Grml release provides fresh software packages from Debian trixie.

Reviewer: ```suggestion
This Grml release brings you fresh software packages from Debian trixie, enhanced hardware support and addresses known bugs from previous releases.
```

> As usual it also incorporates current hardware support and fixes known bugs from the previous Grml release.

Reviewer: ```suggestion
```

> With release version 2024.12 we announce our first release supporting 64-bit ARM CPUs (Architecture `arm64`)!

Reviewer: ```suggestion
With version 2024.12 Grml, for the first time ever, supports 64-bit ARM CPUs (Architecture `arm64`)!
```

> We would like to thank [netcup](https://www.netcup.com/) for their financial support in making this possible.

Reviewer: ```suggestion
This milestone was made possible thanks to the financial support from [netcup](https://www.netcup.com/).
```

> At the same time we want to point out that releases for [32-bit x86 PCs have been discontinued]({{< ref "416-grml32-sunset" >}}).

Reviewer: ```suggestion
As previously announced, releases for [32-bit x86 PCs have been discontinued]({{< ref "416-grml32-sunset" >}}).
```

> With the deprecation of the grml32 flavor, we instead implemented UEFI 32-bit boot support on our Grml amd64 flavor.

Reviewer: ```suggestion
With the grml32 flavor removed, we've introduced UEFI 32-bit boot support on our Grml amd64 flavor to ensure 64-bit PCs with 32-bit Firmware can use this release.
```

> For detailed information about the changes between 2024.02 and 2024.12 have a look at [the official release announcement](https://grml.org/changelogs/README-grml-2024.12/).

Reviewer: ```suggestion
For a detailed overview of the changes from 2024.02 to 2024.12, please check out [the official release announcement](https://grml.org/changelogs/README-grml-2024.12/).
```

Author: ACK, geat suggestions, applied all of them!

> Thanks to everyone contributing to Grml and this release, stay healthy and happy Grml-ing!

Reviewer 2: This feels not soo good too me. What about:

"Thanks to everyone who contributed to Grml and this release, [...]" or
"Thanks to everyone for their contributions to Grml and this release, [...]"
(with the help of DeepL)

Everything else LGTM! Thx!

Author: ack, thx - done
